owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/files/chunks/ChunkedUploadRemoteFileOperation.java

```diff
@@ -95,10 +96,6 @@ protected RemoteOperationResult uploadFile(OwnCloudClient client) throws Excepti
             } else {
                 mPutMethod = new PutMethod(new URL(uriPrefix + File.separator + chunkIndex), mFileRequestBody);
 
-                if (mRequiredEtag != null && mRequiredEtag.length() > 0) {
-                    mPutMethod.addRequestHeader(IF_MATCH_HEADER, "\"" + mRequiredEtag + "\"");
-                }
-
```

Comment:
Should we remove this header? Isn't it mandatory?

After this removal, there are some unused imports. Clean them 👍

Reply:
It is not required to add the If-Match header on each chunk, but it should be in the final move according to other clients: https://github.com/owncloud/android/issues/2752#issuecomment-820381240